File: myKitchenStory/src/main/java/com/kitchenstory/models/ShopMenu.java
package com.kitchenstory.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "products")
public class ShopMenu {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long id;
	private String name;
	private String descr;
	private String imagepath;
	private double price;
	private String status;
	private String code;
	private String createdate;
	private String modifydate;
	
}

File: ngKitchenStorySource/src/app/shopmenu/shopmenu.ts
export class Shopmenu {
    id: number;
    name: string;
    descr: string;
    imagepath: string;
    price: number;
    status: string;
    code: string;    
    createdate: string;
    modifydate: string;
}

export class Basket {
    id: number;
    name: string;
    userid: number;
    price: number;
}

export class OrderPayment {
    payname: string;
    paycardno: number;
    payexpdate:number;
    paythreedigits:number;
    payamount: number;
}
